README.md

```diff
@@ -120,30 +130,29 @@ storyblokInit({
 
 `gatsby-source-storyblok` does three actions when you initialize it:
 
-- Provides a `useStoryblokState` custom hook in your app, that parses story content JSON into the object.
+- Provides a `<StoryblokStory />` that includes `useStoryblokState` that receives a story object.
```

Comment:
`useStoryblokState` is internal and user don't even need to know about it. I'd re-phrase it in this lines: `Provides a <StoryblokStory />` component to enable live-editting experience for a story, automatically.

Reply:
Thanks @alexjoverm ! Fixed it👍